phys2denoise/metrics/chest_belt.py

```diff
@@ -42,13 +47,38 @@ def respiratory_variance_time(resp, peaks, troughs, samplerate, lags=(0, 4, 8, 1
        respiratory-variation-related fluctuations from neuronal-activity-related
        fluctuations in fMRI”, NeuroImage, vol.31, pp. 1536-1548, 2006.
     """
-    timestep = 1 / samplerate
+    if isinstance(data, physio.Physio):
```

Comment:
I was that you repeat this few following lines often in different metrics - if that is the case, I would suggest to make it a function on its own or even better to add it to the decorator you created. If you make it a function, the return can be done within the metric function itself (without the wrapper), if you move this to the decorator (better), you can revert to the original metric functions, and deal with the input and returns of the metrics within the wrapper!